requirements/pip.txt

```diff
@@ -41,7 +41,7 @@ kombu==4.3.0  # pyup: ignore
 # when ALWAYS_EAGER = True
 celery==4.1.1  # pyup: ignore
 
-django-allauth==0.42.0
+django-allauth==0.43.0
```

Comment:
We should double check the `base.html` change described in the changelog.

Reply:
> In previous versions, the ``allauth`` app included a ``base.html``
template. This template could conflict with an equally named template at
project level. Therefore, ``base.html`` has now been moved to
``account/base.html`` -- you will need to check your templates and likely
override ``account/base.html`` within your project.